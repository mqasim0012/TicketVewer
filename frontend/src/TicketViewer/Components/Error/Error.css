.error {
    background-color: rgb(199, 83, 83);
    color: white;
    padding: 20px;
    max-width: 80%;
}