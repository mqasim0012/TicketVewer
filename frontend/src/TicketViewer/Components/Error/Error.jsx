import "./Error.css"

const Error = (props) => {
    return (
        <div>
            Uh oh! Something went a little wrong: <br /><br />
            <div className="error">
                {props.error}: {props.description}
            </div>
        </div>
    )
}

export default Error;