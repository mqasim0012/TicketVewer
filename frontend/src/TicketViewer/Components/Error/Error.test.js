import React from 'react';
import renderer from 'react-test-renderer';
import Error from '.';

test('Error renders correctly', () => {
    const sampleError = renderer.create(
        <Error error="Error:" description="Could not authenticate you" />
    )
    
    const tree = sampleError.toJSON();
    expect(tree).toMatchSnapshot();
})