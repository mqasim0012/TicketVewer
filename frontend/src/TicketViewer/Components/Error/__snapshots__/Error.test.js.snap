// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders correctly 1`] = `
<div>
  Uh oh! Something went a little wrong: 
  <br />
  <br />
  <div
    className="error"
  >
    Error:
    : 
    Could not authenticate you
  </div>
</div>
`;
